docs/rules/no-native-attributes.md

```diff
@@ -0,0 +1,35 @@
+# Disallows use of native attributes as properties (no-native-attributes)
+
+Using global native attributes as lit properties can have unintended effects,
+like for example the native `title` attribute will cause a tiny popup to show up
+over your custom element, and may mess up the accessibility tree for your
+component unintentionally.
+
+## Rule Details
+
+This rule disallows using global native attributes as lit properties.
+
+The following patterns are considered warnings:
+
+```ts
+class MyEl extends LitElement {
+  static properties = {
```

Comment:
just to be consistent with the rest of the docs (getter, no trailing comma):

```ts
static get properties() {
  return {
    title: { type: String },
    role: { type: String }
  };
}
```

same below